templates/rechecks.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'layout.html' %}
+{% block content %}
+<br>
+<h2>Recheck</h2>
+    <form action='/rechecks' method='post'>
+      <textarea id='reportInput' name='recheck' rows='1' cols='60' placeholder='Your Steam ID, i.e: 76561198023727681'></textarea>
+      <p class='help-block'>Login to <b><a href="https://steamcommunity.com/">https://steamcommunity.com/</a></b> to find your unique Steam ID.</p>
```

Comment:
@nibalizer I'll consider merging after you get rid of the steam stuff :P. Just delete line 7 and the `placeholder` in line 6.